src/evaluators/interpolation/PHAL_DOFCellToSide.hpp

```diff
@@ -59,14 +62,66 @@ class DOFCellToSideBase : public PHX::EvaluatorWithBaseImpl<Traits>,
     CELL_VECTOR,
     CELL_TENSOR,
     NODE_SCALAR,
+    NODE_SCALAR_SIDESET,
     NODE_VECTOR,
+    NODE_VECTOR_SIDESET,
     NODE_TENSOR,
-    VERTEX_VECTOR
+    VERTEX_VECTOR,
+    VERTEX_VECTOR_SIDESET
   };
 
   LayoutType layout;
 
   MDFieldMemoizer<Traits> memoizer;
+
+  Kokkos::DynRankView<int, PHX::Device> dimsView;
+
+public:
+
+  typedef Kokkos::View<int***, PHX::Device>::execution_space ExecutionSpace;
+  struct DOFCellToSide_CellScalar_Tag{};
```

Comment:
Another "taste" suggestion: if we remove `DOFCellToSide_` from the tag names, as well as the policy names, we keep the code slimmer and more readable.

Reply:
I agree that would be much nicer to look at it, I'll change these